lib/dndstrings-captions.sty

```diff
@@ -88,38 +88,38 @@
     \renewcommand\sensesname{Sensi}
     \renewcommand\languagesname{Linguaggi}
     \renewcommand\challengename{Sfida}
-%    \newcommand\xpname{XP}
-%    % Monster attack
-%    \renewcommand\ftname{ft.}
-%    \renewcommand\meleeattackname{Melee Weapon~Attack}
-%    \renewcommand\rangedattackname{Ranged Weapon~Attack}
-%    \renewcommand\meleeorrangedattackname{Melee~or~Ranged~Weapon~Attack}
-%    \renewcommand\tohitname{to~hit}
-%    \renewcommand\reachname{reach}
-%    \renewcommand\rangename{range}
-%    \renewcommand\hitname{Hit}
-%    \renewcommand\damagename{damage}
-%    % Spell levels
-%    \renewcommand\spellcantripsname{Cantrips}
-%    \renewcommand\spellfirstlevelname{1st~level}
-%    \renewcommand\spellsecondlevelname{2nd~level}
-%    \renewcommand\spellthirdlevelname{3rd~level}
-%    \renewcommand\spellfourthlevelname{4th~level}
-%    \renewcommand\spellfifthlevelname{5th~level}
-%    \renewcommand\spellsixthlevelname{6th~level}
-%    \renewcommand\spellseventhlevelname{7th~level}
-%    \renewcommand\spelleighthlevelname{8th~level}
-%    \renewcommand\spellninthlevelname{9th~level}
-%    % Spell slots
-%    \renewcommand\spellatwillname{at~will}
-%    \renewcommand\spelloneslotname{1~slot}
-%    \renewcommand\spelltwoslotsname{2~slots}
-%    \renewcommand\spellthreeslotsname{3~slots}
-%    \renewcommand\spellfourslotsname{4~slots}
-%    % Innate spellcasting
-%    \renewcommand\innateatwillname{At~will}
-%    \renewcommand\numberperdayname[1]{#1/day}
-%    \renewcommand\numberperdayeachname[1]{#1/day~each}
+    \newcommand\xpname{PE}
+    % Monster attack
+    \renewcommand\ftname{m.}
+    \renewcommand\meleeattackname{Attacco con Arma~da~Mischia}
```

Comment:
Thanks, @Toiwat. You will want to put tildes (~) in all the spaces. I had missed a few, and #153 puts them in.

Reply:
Done. I did notice some spaces didn't have the tilde, but I thought it was intentional.